backend/src/main/java/com/hooria/todo/domain/Status.java

```diff
@@ -0,0 +1,24 @@
+package com.hooria.todo.domain;
+
+import java.util.Arrays;
+
+public enum Status {
+    NONE(0), TODO(1), IN_PROGRESS(2), DONE(3);
+
+    private final int code;
+
+    Status(int code) {
+        this.code = code;
+    }
+
+    public int getCode() {
+        return code;
+    }
+
+    public static Status of(int code) {
+        return Arrays.stream(Status.values())
+            .filter(s -> s.code == code)
+            .findFirst()
+            .orElse(NONE);
+    }
+}
```

Comment:
```suggestion
public enum Status {

    NONE, TODO, IN_PROGRESS, DONE;
}
```

이렇게만 만들어놓고 뷰 단에서 넘어온 status 정수 값으로 Status.values()[status - 1] 로 접근하는 게 간단명료할 거 같아 😁

Reply:
NONE 은 뭐야?! 필요한 건가?? 😲

Reply:
아 유 지니어스?

Reply:
> NONE 은 뭐야?! 필요한 건가?? 😲

혹시라도 DB나 요청으로 코드값(1~3) 범위 외의 값이 들어올 경우에 Exception을 던져주기보다 그냥 NONE 을 반환해주는게 구현이 편할 것 같아서 요렇게 해보았습니다~

Reply:
NONE 으로 우리가 응답하면 뷰 단에서는 어떤 작업을 해야하는 거야?! 😎